tests/legacy-cli/e2e/setup/500-create-project.ts

```diff
@@ -18,13 +18,20 @@ export default async function() {
   } else {
     const extraArgs = [];
 
-    if (argv['ivy']) {
-      extraArgs.push('--enable-ivy');
-    }
-
     await ng('new', 'test-project', '--skip-install', ...extraArgs);
     await expectFileToExist(join(process.cwd(), 'test-project'));
     process.chdir('./test-project');
+
+    if (!argv['ivy']) {
```

Comment:
At some point we should flip this flag, so that we have to opt-in VE tests instead of opt-in Ivy ones.

Reply:
Good idea. I will do it later today or tmr.

Reply:
As discussed offline, let’s tackle this in a separate PR were we enable the full test suit for Ivy and remove `—ivy` and add `—ve` so that all tests will run in ivy mode by default.